src/main/java/seedu/address/logic/parser/RemindCommandParser.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.util.StringUtil;
+import seedu.address.logic.commands.RemindCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.ReminderDatePredicate;
+
+/**
+ * Parses input argument and creates a new RemindCommand object
+ */
+public class RemindCommandParser implements Parser<RemindCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindCommand
```

Comment:
What exactly does "in the context of the FindCommand" mean?

Reply:
oops, meant to be RemindCommand